app/src/main/java/com/nervousfish/nervousfish/modules/pairing/APairingHandler.java

```diff
@@ -110,30 +112,40 @@ boolean saveContact(final byte[] bytes) {
             contact = (Contact) ois.readObject();
         } catch (final ClassNotFoundException | IOException e) {
             LOGGER.error(" Couldn't start deserialization!");
-            return false;
+            e.printStackTrace();
```

Comment:
Stacktrace should be printed using the logger (applies multiple times in this class)